### PR TITLE
Adding method for creating a unique variable

### PR DESCRIPTION
**Pull request by blishko**

### CODE REVIEW

File: src/logics/Logic.cc [hunk 1]
```diff
@@ -956,6 +958,11 @@ PTRef Logic::mkVar(SRef s, const char* name) {
     return ptr;
 }
 
+PTRef Logic::mkUniqueVar(SRef s) {
+    std::string uniqueName = "@" + std::to_string(uniqueVarCount++);
```
Comment (aehyvari) [inline src/logics/Logic.cc:962]: Put the "@" in some const char near line 65 and use that instead of hardcoding it in the function.

**Review by aehyvari — APPROVED** [review]
Looks good!

File: src/logics/Logic.cc (hunk 1 above)
Comment (blishko) [reply src/logics/Logic.cc]: Fixed.

**Comment by blishko** [comment]
As discussed, the meaning is changing from a unique variable to a unique abstract value (for internal solver use).

**Review by aehyvari — APPROVED** [review]
I think this one's good to go.

**Pull request merged**